other/templates/request.j2

```diff
@@ -1,8 +1,27 @@
 
 async def {{ name.lower_snake_case }}(self{% for param in params %}, {{ param.name.lower_snake_case }}{% endfor %}):
-    """ Generated by dcsdkgen
+    """
```

Comment:
According to https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings there is no newline after `"""`. What's your take on that?

Reply:
I don't care :-). I'll change it.

Reply:
Oh, I see why it is like this. If I don't do that, then the next paragraph is not aligned correctly, and that results in a Python indentation error.

Struggling a bit with jinja2 sometimes :sweat_smile: